subsys/bluetooth/audio/shell/bap_broadcast_assistant.c

```diff
@@ -430,6 +492,183 @@ static int cmd_bap_broadcast_assistant_add_src(const struct shell *sh,
 	return result;
 }
 
+static bool broadcast_source_found(struct bt_data *data, void *user_data)
+{
+	struct bt_bap_broadcast_assistant_add_src_param param = { 0 };
+	const struct bt_le_scan_recv_info *info = user_data;
+	struct bt_uuid_16 adv_uuid;
+	uint32_t broadcast_id;
+	int err;
+
+	/* Verify that it is a BAP broadcaster*/
+
+	if (data->type != BT_DATA_SVC_DATA16) {
+		return true;
+	}
+
+	if (data->data_len < BT_UUID_SIZE_16 + BT_AUDIO_BROADCAST_ID_SIZE) {
+		return true;
+	}
+
+	if (!bt_uuid_create(&adv_uuid.uuid, data->data, BT_UUID_SIZE_16)) {
+		return true;
+	}
+
+	if (bt_uuid_cmp(&adv_uuid.uuid, BT_UUID_BROADCAST_AUDIO) != 0) {
+		return true;
+	}
+
+	broadcast_id = sys_get_le24(data->data + BT_UUID_SIZE_16);
+
+	if (broadcast_id != auto_scan.broadcast_id) {
+		/* Not the one we want */
+
+		return false;
```

Comment:
```suggestion
		return false;
```
Do we need an empty line after a comment? (just asking)

Reply:
Nah :)